src/styles/utils/_variables.scss

```diff
@@ -5,4 +5,8 @@
     --text-white: rgba(255, 255, 255, 1);
     --background-primary: rgba(165, 181, 196, 1);
     --background-secondary: rgba(60, 79, 109, 1);
+    --background-homepage: rgba(217, 216, 221, 1);
+
+    --h1: 28px;
+    --h2: 20px;
```

Comment:
add default font size 16px and set h1, h2, h3, h4 with specific rem sizes from the project